### ISSUE TITLE
The problem of log

### ISSUE THREAD

**Issue by hcscysh** [DepthAnything/Depth-Anything-V2#204]
I train my own dataset, and the log is as folws. The result is very poor, I want to know, Is the network converged?
xFormers not available
xFormers not available
[2024-11-07 16:38:46,640][    INFO] {'bs': 4,
 'dataset': 'mono360',
 'encoder': 'vitb',
 'epochs': 120,
 'img_size': 518,
 'local_rank': 0,
 'lr': 5e-06,
 'max_depth': 20.0,
 'min_depth': 0.001,
 'ngpus': 1,
 'port': 20596,
 'pretrained_from': '../checkpoints/depth_anything_v2_vitb.pth',
 'save_path': 'exp/mono360'}

train.py:98: FutureWarning: You are using `torch.load` with `weights_only=False` (the current default value), which uses the default pickle module implicitly. It is possible to construct malicious pickle data which will execute arbitrary code during unpickling (See https://github.com/pytorch/pytorch/blob/main/SECURITY.md#untrusted-models for more details). In a future release, the default value for `weights_only` will be flipped to `True`. This limits the functions that could be executed during unpickling. Arbitrary objects will no longer be allowed to be loaded via this mode unless they are explicitly allowlisted by the user via `torch.serialization.add_safe_globals`. We recommend you start setting `weights_only=True` for any use case where you don't have full control of the loaded file. Please open an issue on GitHub for any issues related to this experimental feature.
  {k: v for k, v in torch.load(args.pretrained_from, map_location='cpu').items() if 'pretrained' in k},

  0%|          | 0/120 [00:00<?, ?it/s][2024-11-07 16:38:48,502][    INFO] ===========> Epoch: 0/120, d1: 0.000, d2: 0.000, d3: 0.000
[2024-11-07 16:38:48,502][    INFO] ===========> Epoch: 0/120, abs_rel: 100.000, sq_rel: 100.000, rmse: 100.000, rmse_log: 100.000, log10: 100.000, silog: 100.000
/media/gu/d54b9541-2b55-4c75-b059-3006d51983d53/lqc/anaconda3/envs/depth-anything-v2/lib/python3.8/site-packages/torch/autograd/graph.py:769: UserWarning: Grad strides do not match bucket view strides. This may indicate grad was not created according to the gradient layout contract, or that the param's strides changed since DDP was constructed.  This is not an error, but may impair performance.
grad.sizes() = [1, 32, 1, 1], strides() = [32, 1, 32, 32]
bucket_view.sizes() = [1, 32, 1, 1], strides() = [32, 1, 1, 1] (Triggered internally at ../torch/csrc/distributed/c10d/reducer.cpp:327.)
  return Variable._execution_engine.run_backward(  # Calls into the C++ engine to run the backward pass
[2024-11-07 16:39:08,495][    INFO] Iter: 0/286, LR: 0.0000050, Loss: 1.304
[2024-11-07 16:41:37,175][    INFO] Iter: 100/286, LR: 0.0000050, Loss: 0.188
[2024-11-07 16:44:08,574][    INFO] Iter: 200/286, LR: 0.0000050, Loss: 0.199
[2024-11-07 16:46:15,395][    INFO] ==========================================================================================
[2024-11-07 16:46:15,396][    INFO]       d1,       d2,       d3,  abs_rel,   sq_rel,     rmse, rmse_log,    log10,    silog
[2024-11-07 16:46:15,396][    INFO]    0.914,    0.990,    0.996,    0.102,    0.040,    0.392,    0.144,    0.047,    0.133
[2024-11-07 16:46:15,397][    INFO] ==========================================================================================

 99%|█████████▉| 119/120 [13:39:04<06:47, 408.00s/it][2024-11-08 06:17:53,280][    INFO] ===========> Epoch: 119/120, d1: 0.992, d2: 0.998, d3: 0.999
[2024-11-08 06:17:53,280][    INFO] ===========> Epoch: 119/120, abs_rel: 0.038, sq_rel: 0.016, rmse: 0.194, rmse_log: 0.069, log10: 0.015, silog: 0.069
[2024-11-08 06:17:55,392][    INFO] Iter: 0/286, LR: 0.0000001, Loss: 0.052
[2024-11-08 06:20:15,972][    INFO] Iter: 100/286, LR: 0.0000000, Loss: 0.044
[2024-11-08 06:22:36,554][    INFO] Iter: 200/286, LR: 0.0000000, Loss: 0.053
[2024-11-08 06:24:37,148][    INFO] ==========================================================================================
[2024-11-08 06:24:37,148][    INFO]       d1,       d2,       d3,  abs_rel,   sq_rel,     rmse, rmse_log,    log10,    silog
[2024-11-08 06:24:37,149][    INFO]    0.991,    0.997,    0.998,    0.042,    0.026,    0.221,    0.078,    0.017,    0.076
[2024-11-08 06:24:37,149][    INFO] ==========================================================================================

100%|██████████| 120/120 [13:45:53<00:00, 408.12s/it]
100%|██████████| 120/120 [13:45:53<00:00, 412.94s/it]
/media/gu/d54b9541-2b55-4c75-b059-3006d51983d53/lqc/anaconda3/envs/depth-anything-v2/lib/python3.8/site-packages/torch/distributed/launch.py:208: FutureWarning: The module torch.distributed.launch is deprecated
and will be removed in future. Use torchrun.
Note that --use-env is set by default in torchrun.
If your script expects `--local-rank` argument to be set, please
change it to read from `os.environ['LOCAL_RANK']` instead. See 
https://pytorch.org/docs/stable/distributed.html#launch-utility for 
further instructions

  main()
